homework_8_os_and_sys_modules/sort.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+import argparse
+
+# Make container for argument specifications
+parser = argparse.ArgumentParser(description='Sorts the contents of a text file, line by line')
+parser.add_argument('file', default=True, help='input file')
+args = parser.parse_args()
+
+
+# Main function
+def read_file_and_sort(file):
+    l_of_lines = []
+    with open(file, 'r') as inf:  # read the file line by line
+        for line in inf:
+            l_of_lines.append(line[:-1])

```

Comment:
Удалять перенос строки срезом - плохой и ненадёжный вариант
```suggestion
            l_of_lines.append(line.strip())

```